src/client/src/containers/Footer/index.tsx

```diff
@@ -202,30 +202,28 @@ class Footer extends React.Component<Props> {
               >
                 <FormattedMessage id="footer.back" defaultMessage="Back" />
               </Link>
-              <Link
-                tabIndex={
-                  !isValidNameAndProjectPath || this.isReviewAndGenerate()
-                    ? -1
-                    : 0
-                }
-                className={classnames(styles.button, {
-                  [buttonStyles.buttonDark]:
-                    this.isReviewAndGenerate() || !isValidNameAndProjectPath,
-                  [buttonStyles.buttonHighlightedBorder]: !this.isReviewAndGenerate(),
-                  [styles.disabledOverlay]:
-                    !isValidNameAndProjectPath || this.isReviewAndGenerate()
-                })}
-                onClick={event => {
-                  this.handleLinkClick(event, pathname);
-                }}
-                to={
-                  pathname === ROUTES.REVIEW_AND_GENERATE
-                    ? ROUTES.REVIEW_AND_GENERATE
-                    : pathsNext[pathname]
-                }
-              >
-                <FormattedMessage id="footer.next" defaultMessage="Next" />
-              </Link>
+              { pathname !== ROUTES.REVIEW_AND_GENERATE &&
+                <Link
+                  tabIndex={
+                    !isValidNameAndProjectPath ? -1 : 0
+                  }
+                  className={classnames(styles.button, buttonStyles.buttonHighlightedBorder, {
+                    [buttonStyles.buttonDark]: !isValidNameAndProjectPath,
+                    [styles.disabledOverlay]:
+                      !isValidNameAndProjectPath
+                  })}
+                  onClick={event => {
+                    this.handleLinkClick(event, pathname);
+                  }}
+                  to={
+                    pathname === ROUTES.REVIEW_AND_GENERATE
```

Comment:
This is never true, right?

Reply:
Yeah, it was actually not necessary earlier as well as the button was disabled in the 'Review and Generate' page. I will remove it now.

Reply:
This is still missing a change right?